src/components/ui/Loader.tsx

```
@@ -0,0 +1,43 @@
+);
+
+type LoaderVariantProps = VariantProps<typeof loaderVariants>;
+
+export interface LoaderProps extends LoaderVariantProps {
+  className?: string;
+}
+
+export const Loader: React.FC<LoaderProps> = ({ size, className }) => {
+  return (
+    <span
+      role="status"
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)